.github/workflows/links.yml

```diff
@@ -0,0 +1,25 @@
+name: Links
+
+on:
+  push:
+  pull_request:
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+    - cron: "00 18 * * *"
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Link Checker
+        id: checker
+        uses: lycheeverse/lychee-action@v1.0.9
```

Comment:
I suggest updating to the latest, a few new versions came out. The latest at this time is v1.1.1